src/com.frostycoolslug.goxlr-utility.sdPlugin/app/display-volume.js

```diff
@@ -24,20 +23,18 @@ displayVolume.onWillDisappear(({action, event, context, device, payload}) => {
     delete volumeMonitors[context];
 });
 
-
 function createVolumeMonitor(context, settings) {
     let serial = settings.serial;
     let mix = settings.mix;
     let channel_a = ChannelNameReadable[settings.channel];
     let channel_b = ChannelNameReadable[settings['sub-channel']];
+    let hide_name = settings.hide_name;
 
     if (volumeMonitors[context] !== undefined) {
-        if (!volumeMonitors[context].equal(context, serial, mix, channel_a, channel_b)) {
```

Comment:
I ended up getting rid of this line completely. It checks if the selected action is different in any way, and doesn't rebuild it if it is the exact same, I couldn't find a reason to avoid re-rendering it as the createVolumeMonitor function won't run very often.

Reply:
Yep, that's reasonable, this isn't going to be triggered often, and even when it is it'll be incredibly quick to update.